new/rfc-for-hugs.md

```diff
@@ -0,0 +1,50 @@
+---
+RFC: unassigned
+Author: Aaron Kalin and JJ Asghar
+Status: Draft
+Type: Informational, Process
+---
+
+# Hugging in the Chef Community
+
+In order to bridge the cultural gap between the US and Worldwide community the concept of hugging should be explained.
+
+## Motivation
+
+    As a Chef Community Participant,
+    I want to express positive emotion to or for another Chef Community Participant,
+    so that we can express our friendship physically.
+
+## Specification
+
+When approaching another Chef Community Member that you wish to engage in a hug. In order to comply with the already
+existing Chef Community Guidelines and Code of Conduct at various community events. You must seek permission from the
+other community member by either verbally or physically indicating that you wish to hug them.
+
+An example, but not limited to, of a verbal indication for permission to hug is to ask "Can I give you a hug?" or "Can we hug?"
+The other party will then respond with verbal consent; for example "Yes, we can" or "Yes, you may."
```

Comment:
Or at least "will" should be changed to "may" - "will" infers that the other party is always going to accept. Something I wish to be the case, but it not universally true.